### ISSUE TITLE
TRACE causes browser to crash

### ISSUE THREAD

**Issue by cj495840252** [DioxusLabs/example-projects#46]
**Problem**

When I use `trunk serve` to start this project. The browser runs normally for a period of time before opening. but when i don't click browser a half minute, my browser will stuck. code doesn't panic
I set `Level::INFO` is useless

```
    console_error_panic_hook::set_once();
    
    let config = WASMLayerConfigBuilder::new()
        .set_max_level(Level::TRACE)
        .build();
    tracing_wasm::set_as_global_default_with_config(config);
    dioxus_web::launch(app);
```

**Screenshots**
![1708763137310](https://github.com/DioxusLabs/example-projects/assets/63491689/bc8f7d7d-0a02-4c0e-9824-b04a6f9ee8b4)

**Environment:**
 - dioxus version: 0.43
 - dioxus-web version: 0.43
 - console_error_panic_hook = "0.1"
 - tracing = "0.1"
 - tracing-wasm = "0.2"
 - wasm-bindgen = {version = "0.2", features = ["enable-interning"] }
 - Rust version: 1.76
 - OS info: MacOS 14.2.1
 - App platform: web

**Comment by ealmloff** [comment]
That issue was fixed in https://github.com/DioxusLabs/dioxus/pull/1925 and released in the dioxus 0.5 pre release. You can update your dioxus version or work around the issue by either:
- Running with the `--hot-reload` flag: `dx serve --hot-reload`
- Or building in release mode: `dx serve --release` (any profile without debug assertions turns off hot reloading)

**Comment by tkr-sh** [comment]
Maybe close it @ealmloff ?
